requirements/edx/base.txt

```diff
@@ -39,10 +39,6 @@ elasticsearch==0.4.5
 facebook-sdk==0.4.0
 feedparser==5.1.3
 firebase-token-generator==1.3.2
-# Master pyfs has a bug working with VPC auth. This is a fix. We should switch
-# back to master when and if this fix is merged back.
-# fs==0.4.0
```

Comment:
FWIW They've moved to github since google code is getting shutdown: https://github.com/PyFilesystem/pyfilesystem